target/scripts/helpers/index.sh

```diff
@@ -1,16 +1,25 @@
 #! /bin/bash
+
 # shellcheck source-path=target/scripts/helpers
 # This file serves as a single import for all helpers
 
+export CHKSUM_FILE=/tmp/docker-mailserver-config-chksum
+
 function _import_scripts
 {
   local PATH_TO_SCRIPTS='/usr/local/bin/helpers'
 
-  . "${PATH_TO_SCRIPTS}/postfix.sh"
-  . "${PATH_TO_SCRIPTS}/accounts.sh"
-  . "${PATH_TO_SCRIPTS}/aliases.sh"
-  . "${PATH_TO_SCRIPTS}/relay.sh"
-  . "${PATH_TO_SCRIPTS}/sasl.sh"
-  . "${PATH_TO_SCRIPTS}/ssl.sh"
+  source "${PATH_TO_SCRIPTS}/accounts.sh"
+  source "${PATH_TO_SCRIPTS}/aliases.sh"
+  source "${PATH_TO_SCRIPTS}/dns.sh"
+  source "${PATH_TO_SCRIPTS}/exit.sh"
+  source "${PATH_TO_SCRIPTS}/log.sh"
+  source "${PATH_TO_SCRIPTS}/miscellaneous.sh"
+  source "${PATH_TO_SCRIPTS}/postfix.sh"
+  source "${PATH_TO_SCRIPTS}/relay.sh"
+  source "${PATH_TO_SCRIPTS}/sasl.sh"
+  source "${PATH_TO_SCRIPTS}/ssl.sh"
+  source "${PATH_TO_SCRIPTS}/tcp_ip.sh"
```

Comment:
What do you think about `network.sh`?

Reply:
Yes, that's a better name. I will commit this in batch later.